### PR TITLE
Angular standalone pager

### LINKED ISSUE #1356: [Pager] styling for numeric buttons
**Is your feature request related to a problem? Please describe.**
After a discussion with @joneff and @tsvetomir we decided to finally rely on real buttons as opposed to Anchor or Span elements in the new Pager component. Thus the numeric buttons that represent the page numbers in the pager will need styling that will remove the "button" look and feel and keep the styling known from the Grid pager.

**Describe the solution you would like to see implemented**
The Button elements that are within **.k-pager-numbers-wrap .k-pager-numbers li** need to be styled accordingly. Here is the exact DOM structure:

```
// within a **.k-pager-numbers-wrap** parent
<ul [ngClass]="{'k-pager-numbers': true, 'k-reset': true}">
            <li *ngIf="start > 1">
                <button
                    type="button"
                    class="k-link"
                    [attr.aria-label]="pageLabel(start - 1)"
                    (click)="changePage(start - 2)">...</button>
            </li>
            <li *ngFor="let num of buttons">
                <button
                    type="button"
                    [attr.aria-label]="pageLabel(num)"
                    [ngClass]="{'k-link': true, 'k-state-selected':currentPage == num}"
                    (click)="changePage(num - 1)">
                    {{num}}
                </button>
            </li>
            <li *ngIf="end < totalPages">
                <button
                    type="button"
                    class="k-link"
                    [attr.aria-label]="pageLabel(end + 1)"
                    (click)="changePage(end)">...</button>
            </li>
        </ul>
```

### LINKED ISSUE #1386: [Pager] Add styles for standalone pager component
Related to:
https://github.com/telerik/kendo-ux-private/issues/98

### PR DESCRIPTION
**Pull request by inikolova**
Related issues:
* fixes #1386
* fixes #1356
* telerik/kendo-ux-private#98

### CODE REVIEW

File: packages/default/scss/pager/_layout.scss [hunk 1]
```diff
@@ -199,9 +201,6 @@
         }
     }
 
-
-
-
     // Responsive
     .k-pager-lg,
```
Comment (PreslavKozovski) [inline packages/default/scss/pager/_layout.scss]: There are still styles for k-pager-lg, based on the design and our conversations, we no longer need these styles, please remove them.

File: packages/default/scss/pager/_layout.scss [hunk 2]
```diff
@@ -199,9 +201,6 @@
...
     .k-pager-md,
```
Comment (PreslavKozovski) [inline packages/default/scss/pager/_layout.scss]: I believe these styles are now relevant only to "-sm" page, we should remove the "-md" selector, as it gives margins to the "k-pager-numbers-wrap", etc.

File: packages/default/scss/pager/_layout.scss [hunk 3]
```diff
@@ -238,13 +231,20 @@
 
     }
 
+    // Show native select
+    .k-pager-sm {
```
Comment (PreslavKozovski) [inline packages/default/scss/pager/_layout.scss]: No need to extract these styles after removing the above selectors.

**Comment by Xizario** [comment]
@inikolova As we already have standalone pager from the react side, prior this PR.
https://www.telerik.com/kendo-react-ui/components/datatools/pager/
Could you take a look at the rendering, and let us know if we need to change something?

**Comment by inikolova** [comment]
OK, I will update the visual tests

**Pull request merged**